grow-website-gatsby/src/pages/index.tsx

```diff
@@ -51,11 +52,15 @@ export const pageQuery = graphql`
 export default class IndexPage extends React.Component<IndexPageProps, {}> {
   public render() {
     return (
-      <div>
-        <div className={styles.gutters}>
+      <Fragment>
+        <Helmet>
+          <html lang="en-NZ" />
+          <title>Grow &endash; Empowering women to thrive in tech</title>
+        </Helmet>
+        <header className={styles.gutters}>
           <div className={styles.header}>
             <div className={styles.headerLogo}>
-              <img src={growLogo} alt="Grow" />
+              <img src={growLogo} alt="The Grow team" />
```

Comment:
I think we want this alt text on the banner image rather than the logo.

Reply:
Whoops, yes that was the intent! 😬

Reply:
@catepalmer Fixed :)